auth2.php

```diff
@@ -0,0 +1,72 @@
+<?php
```

Comment:
Why is their two versions of auth.php ?

Reply:
boilerplate code snuck in...will remove, thanks!